api/tsconfig.json

```diff
@@ -0,0 +1,110 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
```

Comment:
We should have this cleaned up as part of this PR. i.e., get rid of all the comments. To start, it should extend the tsconfig from the root directory

Reply:
I can extend the one from our root directory, but I don't think keeping these commented out options is a bad idea